src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/tests/latest/test_redis_scenario.py

```diff
@@ -13,21 +13,34 @@ def test_redis_cache(self, resource_group):
 
         self.kwargs = {
             'rg': resource_group,
-            'loc': 'WestUS',
+            'loc': 'WestUS2',
```

Comment:
You should check the result of applicable commands. It might help to create multiple scenario tests.

resource/tests/latest/test_resource.py has examples of testing commands that receive json input. See test_subscription_level_deployment